.github/workflows/main.yml

```diff
@@ -99,6 +99,8 @@ jobs:
 
       - name: Checkout source files
         uses: actions/checkout@v2
+        with:
+          submodules: 'true'
```

Comment:
This fixes the following:
![image](https://user-images.githubusercontent.com/5289520/127423312-45408d98-607d-442f-80be-4dc22d8c9c65.png)

(missing submodules)